doc/source/examples/dem/sandpile-formation/sandpile-formation.rst

```diff
@@ -0,0 +1,310 @@
+We expect that the Elastic-Plastic Spring-Dashpot model will give closer results to the height and angle from the experiment than the viscous model or the constant torque model. Indeed, the constant torque model produces an oscillation that can prevent the pile from reaching a stable state and the viscous model is not adapted here as the viscous effects are not major.
+
+--------------
+Parameter File
+--------------
+
+Mesh
+~~~~
+
+The mesh is a hopper with 50.5° angle generated with GMSH, with a channel connecting the hopper to the part with the flat surface.
+The geometry follows the one used by Zuriguel *et al.* [#Zuriguel2007]_ and was handled in order to generate a structured mesh.
+
+.. code-block:: text
+
+    subsection mesh
+      set type                = gmsh
+      set file name           = ./sandpile.msh
+      set check diamond cells = true
+      set initial refinement  = 1
+    end
+
+
+.. figure:: images/mesh-sandpile.png
+    :width: 500
+    :alt: Mesh
+    :align: center
+
+    2D mesh of the hopper
+
+
+.. note::
+
+  The mesh can be generated using the following line:
+
+   .. code-block:: bash
+      :class: copy-button
+
+      gmsh -2 sandpile.geo
+
+
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The 3000 particles are bi-dispersed, with 2500 having a diameter of 6.9 mm and 500 having a diameter of 8.9 mm.
+
+The following properties are chosen according to the J. Ai *et al.* paper:
+
+* Custom distribution of disks with 70% of disks with a 6.9 mm diameter and 30% with an 8.9 mm diameter
+* Density (1056 :math:`kg/m^3`)
+* Young's modulus (4 MPa)
+* Poisson's ratio (0.49)
+* Friction coefficient of particle-particle (0.8)
+* Rolling resistance coefficient of particle-particle (0.3)
+
+The same properties were given to the wall as they were not specified.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+      set g                        = 0.0, -9.81
+      set number of particle types = 1
+      subsection particle type 0
+        set size distribution type            = custom
+        set custom diameters                  = 0.0069 , 0.0089
+        set custom volume fractions           = 0.7   , 0.3
+        set number of particles               = 3000
+        set density particles                 = 1056
+        set young modulus particles           = 4000000
+        set poisson ratio particles           = 0.49
+        set restitution coefficient particles = 0.7
+        set friction coefficient particles    = 0.8
+        set rolling friction particles        = 0.3
+      end
+      set young modulus wall           = 4000000
+      set poisson ratio wall           = 0.49
+      set restitution coefficient wall = 0.7
+      set friction coefficient wall    = 0.8
+      set rolling friction wall        = 0.3
+    end
+
+.. note::
+
+  Only the value of the restitution coefficient was not given in the paper.
```

Comment:
```suggestion
  Only the value of the restitution coefficient was not given in the reference paper.
```

Do  you have a rationale for the value you chose? Just a quick indication. Otherwise, if it did not have an impact, just state it :).